src/convert.cpp

```diff
@@ -626,7 +616,12 @@ void Converter::cnvExifDate(const char* from, const char* to) {
     return;
   if (!prepareXmpTarget(to))
     return;
-  int year = 0, month = 0, day = 0, hour = 0, min = 0, sec = 0;
+  int year = 0;
```

Comment:
I see that you normally use the initialization with curly brackets : `int year{0}`. Why not here?

Reply:
That's done in member variables